test/src/test/kotlin/graphql/nadel/tests/next/fixtures/introspection/NamespaceLikeShortCircuitsIntrospectionTestSnapshot.kt

```diff
@@ -0,0 +1,50 @@
+// @formatter:off
+package graphql.nadel.tests.next.fixtures.introspection
+
+import graphql.nadel.tests.next.ExpectedNadelResult
+import graphql.nadel.tests.next.ExpectedServiceCall
+import graphql.nadel.tests.next.TestSnapshot
+import graphql.nadel.tests.next.listOfJsonStrings
+import kotlin.Suppress
+import kotlin.collections.List
+import kotlin.collections.listOf
+
+private suspend fun main() {
+    graphql.nadel.tests.next.update<NamespaceLikeShortCircuitsIntrospectionTest>()
+}
+
+/**
+ * This class is generated. Do NOT modify.
+ *
+ * Refer to [graphql.nadel.tests.next.UpdateTestSnapshots]
+ */
+@Suppress("unused")
+public class NamespaceLikeShortCircuitsIntrospectionTestSnapshot : TestSnapshot() {
+    override val calls: List<ExpectedServiceCall> = listOf(
+            )
```

Comment:
So here the calls are empty because the tests are executed by our internal introspection service.